src/library/export/trackexportworker.cpp

```diff
@@ -26,10 +26,13 @@ QMap<QString, mixxx::FileInfo> createCopylist(const TrackPointerList& tracks) {
     // efficiently.
     QMap<QString, mixxx::FileInfo> copylist;
     for (const auto& pTrack : tracks) {
+        VERIFY_OR_DEBUG_ASSERT(pTrack != nullptr) {
+            continue;
+        }
         auto fileInfo = pTrack->getFileInfo();
         if (fileInfo.resolveCanonicalLocation().isEmpty()) {
             qWarning()
-                    << "File not found or inaccessible while exporting"
+                    << "TrackExportWorker: File not found or inaccessible while exporting"
```

Comment:
wbout a QLoggingCategory instead of the prefix?

Reply:
sure. I decided on mixxx::Logger though